test/functional/tool_utils.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+# Copyright 2014 BitPay Inc.
+# Copyright 2016-present The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or https://opensource.org/license/mit.
+"""Exercise the utils via json-defined tests."""
+
+from test_framework.test_framework import BitcoinTestFramework
+
+import difflib
+import json
+import logging
+import os
+import subprocess
+from pathlib import Path
+
+
+class ToolUtils(BitcoinTestFramework):
+    def set_test_params(self):
+        self.num_nodes = 0  # No node/datadir needed
+
+    def setup_network(self):
+        pass
+
```

Comment:
```suggestion

   def skip_test_if_missing_module(self):
      self.skip_if_no_bitcoin_util()

```
NIT: Test will fail when project is build without util `cmake -B build -DBUILD_UTIL=OFF` add test to see if test needs to be skipped.